tests/hotplug/affinity.go

```diff
@@ -30,7 +30,7 @@ import (
 	"kubevirt.io/kubevirt/tests/testsuite"
 )
 
-var _ = Describe("[sig-compute]VM Affinity", decorators.SigCompute, decorators.SigComputeMigrations, decorators.RequiresTwoSchedulableNodes, decorators.VMLiveUpdateFeaturesGate, Serial, func() {
```

Comment:
While `VMLiveUpdateFeaturesGate` doesn't represent the feature anymore can we just rename the decorator?

Reply:
What for though? The tests would always be run now.

Reply:
Ok, makes sense, I'll re-add it